catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -461,15 +461,24 @@
 
     <bean id="queryWorkspaceService"
           class="org.codice.ddf.catalog.ui.query.monitor.impl.WorkspaceServiceImpl">
+        <cm:managed-properties
+                persistent-id="org.codice.ddf.catalog.ui.query.monitor.impl.WorkspaceServiceImpl"
+                update-strategy="container-managed"/>
         <argument ref="catalogFramework"/>
-        <argument ref="filterBuilder"/>
         <argument ref="workspaceTransformer"/>
         <argument>
-            <bean class="org.codice.ddf.catalog.ui.query.monitor.impl.FilterServiceImpl">
+            <bean class="org.codice.ddf.catalog.ui.query.monitor.impl.WorkspaceQueryBuilderImpl">
                 <argument ref="filterBuilder"/>
+                <argument>
+                    <bean class="org.codice.ddf.catalog.ui.query.monitor.impl.FilterServiceImpl">
+                        <argument ref="filterBuilder"/>
+                    </bean>
+                </argument>
             </bean>
         </argument>
         <argument ref="securityService"/>
+        <argument ref="persistentStore"/>
+        <argument value="100"/>
```

Comment:
If this is configurable via admin UI should there be a `<property>` tag for it also?

Reply:
It is configurable (see `/projects/DdfMaster3/catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.query.monitor.impl.xml`). There doesn't need to be a property tag.

Reply:
But if it's configurable from the admin ui, how is it updated without a getter / setter?  Do we want to be able to change it after the service is instantiated?

Reply:
oh, i see... let me fix that :-)

Reply:
wait, there is a setter on line 107

Reply:
i think it needs to be a property then, so it gets called from blueprint/metatype when the configuration is changed